[Manifest ID article] Fix future-looking language and add constraints (#7524)
GoogleChrome/developer.chrome.com: site/en/articles/pwa-manifest-id/index.md

authors:
  - petelepage
date: 2021-09-16
updated: 2023-10-12
description: |

>   There's a new optional `id` property as part of the web app manifest spec,

Reviewer: ```suggestion
  The optional `id` property is part of the web app manifest spec,
```

>   that allows you to explicitly define the identifier used for your PWA.

Reviewer: ```suggestion
  and allows you to explicitly define the identifier used for your PWA.
```
